pkgs/development/python-modules/narwhals/default.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  hatchling,
+}:
+
+buildPythonPackage rec {
+  pname = "narwhals";
+  version = "1.9.2";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
```

It's generally recommended to avoid using `rec`

```suggestion
let
  pname = "narwhals";
  version = "1.9.2";
in
buildPythonPackage {
  pyproject = true;

  disabled = pythonOlder "3.8";

  src = fetchPypi {
    inherit pname version;
```

---

Thanks!